Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -705,32 +704,32 @@ void ClearInterrupt(DIInterruptType interrupt)
 }
 
 // Checks the drive state to make sure a read-like command can be performed.
-// If false is returned, SetHighError will have been called, and the caller
+// If false is returned, SetDriveError will have been called, and the caller
 // should issue a DEINT interrupt.
 static bool CheckReadPreconditions()
 {
-  if (!IsDiscInside())  // Implies ERROR_COVER or ERROR_NO_DISK
+  if (!IsDiscInside())  // Implies CoverOpened or NoMediumPresent
```

Comment:
```suggestion
  if (!IsDiscInside())  // Implies CoverOpened or MediumNotPresent
```

Reply:
This one is correct; it refers to `DriveState::NoMediumPresent`, not `DriveError::MediumNotPresent`.